portal/config/eproms/Organization.json

```diff
@@ -2164,7 +2169,7 @@
         }
       ],
       "id": 28000,
-      "language": "pt_BR",
+      "language": "en_US",
```

Comment:
I think this should probably be be added as an option (currently just the pre-selected default for `id=28000`/`Brazil (Region/Country Site)`), by adding it to the `extension` array.

Reply:
I also assume we don't want to enable `pt_BR` region-wide (`id=28000`/`Brazil (Region/Country Site)`)

Reply:
I *think* what I have is correct here...
PCCTC's aim is to have all Brazilian orgs the same for language: english as the default, with portugese optional. It's a goal, but not an absolute - at some point there may be a brazilian org that wants it otherwise.
Our implementation of org config inheritance is opaque to me, @pbugni  and I have chatted about it a bit. But I think this change is correct & errs on the safe side.